### PR TITLE
Allow for Subtitles to be added to embedded videos

### PR DESCRIPTION
**Pull request by marcwalsh**
This allows for subtitles to be added to videos, but adding a variable for the TRACK tag in videowrapper.html.

Also adding subtitle file for the Open Canvas video - starting to solve #81.

@acabunoc could you review and test this for me please?

### CODE REVIEW

File: subs/OpenCanvas-Subs-EN.vtt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+00:01:29.751 --> 00:01:34.000
+ - So if you're thinking about ways outsiders can become insiders, the user channel and
+
+24
+00:01:34.001 --> 00:01:38.000
+ - contributor channel, on the right hand side, will help you plan pathways for others to get involved.
+
+25
+00:01:38.001 --> 00:01:44.000
+ - So that's it! Get started! All you need to is print this off, get a pen fill it out.
```
Comment (abbycabs) [inline subs/OpenCanvas-Subs-EN.vtt:103]: I say 'So' a lot...

File: _includes/videowrapper.html [hunk 2]
```diff
@@ -2,4 +2,5 @@
 
 <video width="100%" height="auto" controls>
     <source src="{{ include.url }}" type="video/mp4">
-</video>
\ No newline at end of file
+    <track src="{{ include.subs }}" kind="subtitles" srclang="en" label="English">
```
Comment (abbycabs) [inline _includes/videowrapper.html]:
```suggestion
    <track src="{{ site.baseurl }}/{{ include.subs }}" kind="subtitles" srclang="en" label="English">
```

File: _articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas.md [hunk 3]
```diff
@@ -32,6 +32,7 @@ Here’s more about the Open Canvas tool from its creator, Abby Cabunoc Mayes, D
 
 {% include videowrapper.html
   url="https://www.filepicker.io/api/file/LZtI0qVRTDqy0stDOkxX"
+  subs="https://mozilla.github.io/open-leadership-training-series/subs/OpenCanvas-Subs-EN.vtt"
```
Comment (abbycabs) [inline _articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas.md]:
```suggestion
  subs="/subs/OpenCanvas-Subs-EN.vtt"
```

File: _includes/videowrapper.html (hunk 2 above)
Comment (abbycabs) [reply _includes/videowrapper.html]: Adding this here removes the need to provide the full path for each video AND it lets us test it locally beforehand

File: _articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas.md (hunk 3 above)
Comment (abbycabs) [reply _articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas.md]: Removed the full path and moved to videowrapper

**Review by abbycabs — CHANGES_REQUESTED** [review]

File: subs/OpenCanvas-Subs-EN.vtt (hunk 1 above)
Comment (marcwalsh) [reply subs/OpenCanvas-Subs-EN.vtt:103]: I'm always the same whenever I type up a conversation I've had!

**Comment by marcwalsh** [comment]
Thanks @acabunoc - I've added the changes you've requested!

Hopefully this is good to merge

**Pull request merged**